src/components/Icon.tsx

```diff
@@ -344,6 +344,9 @@ export const Icons = {
   house: (
     <path d="M5.00299 22H9.00299H15.003H19.003C20.106 22 21.003 21.103 21.003 20V11C21.003 10.735 20.898 10.48 20.71 10.293L12.71 2.29301C12.319 1.90201 11.687 1.90201 11.296 2.29301L3.29599 10.293C3.10799 10.48 3.00299 10.735 3.00299 11V20C3.00299 21.103 3.89999 22 5.00299 22ZM10.003 20V15H14.003V20H10.003ZM5.00299 11.414L12.003 4.41401L19.003 11.414L19.004 20H16.003V15C16.003 13.897 15.106 13 14.003 13H10.003C8.89999 13 8.00299 13.897 8.00299 15V20H5.00299V11.414Z" />
   ),
+  bell: (
+    <path d="M16 11.586V8C16 4.783 13.815 2.073 10.855 1.258C10.562 0.52 9.846 0 9 0C8.154 0 7.438 0.52 7.145 1.258C4.185 2.074 2 4.783 2 8V11.586L0.293 13.293C0.105 13.48 0 13.734 0 14V16C0 16.553 0.447 17 1 17H17C17.553 17 18 16.553 18 16V14C18 13.734 17.895 13.48 17.707 13.293L16 11.586ZM16 15H2V14.414L3.707 12.707C3.895 12.52 4 12.266 4 12V8C4 5.243 6.243 3 9 3C11.757 3 14 5.243 14 8V12C14 12.266 14.105 12.52 14.293 12.707L16 14.414V15ZM9 20C10.311 20 11.407 19.166 11.818 18H6.182C6.593 19.166 7.689 20 9 20Z" />
```

Comment:
This icon doesn't appear to have been copied to properly sit within a 24x24 bounding box. See the result here:
<img width="58" alt="Screen Shot 2022-09-14 at 3 07 18 PM" src="https://user-images.githubusercontent.com/1143861/190241459-85c7001a-cf95-4df1-a733-04af6af1b793.png">
This causes the button to not align as expected against text.

Reply:
Huh, you're right, I'll check it